census/src/main/java/io/grpc/census/CensusTracingModule.java

```diff
@@ -315,6 +315,12 @@ public void streamCreated(Attributes transportAtts, Metadata headers) {
         headers.discardAll(tracingHeader);
         headers.put(tracingHeader, span.getContext());
       }
+      span.addAnnotation("Real stream created.");
```

Comment:
I see in the bug you said this would always be done. I thought this would be conditional if createPendingStream() was called. Mark did document it would be conditional at b/251881969#comment2

Reply:
oh i see, Mark said if we queue then we add annotation.
I see we can add a boolean if createPendingStream is called and then conditionally add annotation for real stream creation time.